Ports/jfduke3d/package.sh

```diff
@@ -6,6 +6,7 @@ install_dir='/usr/local/share/games/jfduke3d'
 launcher_name='Duke Nukem 3D'
 launcher_category='Games'
 launcher_command="${install_dir}/duke3d"
+icon_file="${SERENITY_INSTALL_ROOT}${install_dir}/game_icon.ico"
```

Comment:
You don't need to copy the icon file into the filesystem; providing an `icon_file` will add the right sections to the binary pointed at by `launcher_command`.